### ISSUE TITLE
Create `related_urls` field in category.json?

### ISSUE THREAD

**Issue by Daniel-at-github** [pyvideo/data#442]
I think that a `related_urls` field in category.json is needed for any url traversal to all the talks in an event. Examples:
* Event schedule
* Event web
* Event social media url's

This `related_urls` could be displayed in all talks of the event.

**Comment by jtemporal** [comment]
Yes! some events keep their website alive even after it was over. I think it is a valid event metadata to have.
